docs/user-guide.md

```diff
@@ -1154,31 +1157,32 @@ Found p6880880_122010_Linux-x86-64.zip : OPatch Utility
 
 ## Prerequisite configuration
 
-Before you run the tool you need to create JSON formatted configuration files
-for the data mount devices and the ASM disk group.
+Before you run the tool you need to create JSON formatted configurations
```

Comment:
I think we can make this more succinct:

"Create JSON formatted configurations for the data mount devices and the ASM disk group."